prediction/src/tests/test_endpoints.py

```diff
@@ -4,13 +4,13 @@
 
     Provides unit tests for the API endpoints.
 """
+from flask import url_for
 from functools import partial
 import json
-
 import pytest
-
-from flask import url_for
-from src.factory import create_app
+import sys
+sys.path.append('.')
+from src.factory import create_app  # NOQA
```

Comment:
Hey @pjbull thanks for your feedback!
I tried out your suggestions but that leads to
1) `ModuleNotFoundError: No module named 'app.src.tests.factory'`
2) `ModuleNotFoundError: No module named 'prediction'`

I'm not that familiar with `pytest` but I can do some research about that.